src/verilog/verilog_typecheck_expr.cpp

```diff
@@ -1592,6 +1592,16 @@ void verilog_typecheck_exprt::implicit_typecast(
       return;
     }
   }
+#if 0
+  else if(src_type.id() == ID_natural)
+  {
+    if(dest_type.id()==ID_integer)
+    {
+      expr = typecast_exprt{expr, dest_type};
+      return;
+    }
+  }
+#endif
```

Comment:
Could a comment please be included to explain why this is defined away?

Reply:
removed the `#if 0`